Dockerfile.node-cache

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 
 FROM ARG_FROM
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends \
     iproute2 \
     iptables \
+&& update-alternatives --set iptables /usr/sbin/iptables-legacy \
```

Comment:
can we add a comment at the top explaining why this is needed? Maybe add a link to https://bugzilla.netfilter.org/show_bug.cgi?id=1422?

Reply:
done

Reply:
now really pushed the modifications